components/conversations/Item.tsx

```diff
@@ -61,6 +60,24 @@ const ConversationsItem: React.FC<ConversationsItemProps> = (props) => {
     }
   };
 
+  // ============================ Menu ============================
+
+  const { trigger, items = [], ...menuOther } = menu || {};
+
+  const menusTriggerNode = useCallback(
+    (value: Conversation) => {
+      if (trigger) {
+        if (typeof trigger === 'function') {
+          const triggerNode = trigger(value);
+          if (React.isValidElement(triggerNode)) return triggerNode;
+        }
+        if (React.isValidElement(trigger)) return trigger;
```

Comment:
```suggestion
        if (React.isValidElement(trigger)) {
          return trigger;
        }
```

return 始终换行，这样可以避免测试覆盖率漏掉。